<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seatadj</class>
 <widget class="QWidget" name="Seatadj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 174, 86);</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 156, 255);</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>62</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 209, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="adj_uplabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>101</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="down_btn">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>208</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="up_btn">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>128</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 255, 227);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="adj_leftlabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="right_btn">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>490</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_btn">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>410</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 255, 227);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000a1;&quot;&gt;SIDE WARDS CONTROLS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>66</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 209, 200);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000a1;&quot;&gt;UP WARD CONTROLS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
